samples/matter/common/src/ota_util.cpp

```diff
@@ -21,6 +21,21 @@ DefaultOTARequestorDriver sOTARequestorDriver;
 chip::BDXDownloader sBDXDownloader;
 chip::DefaultOTARequestor sOTARequestor;
 
+/**
+ * Select recommended OTA image processor implementation.
+ *
+ * If the application uses QSPI external flash and enables API for controlling
+ * power states of peripherals, select the implementation that automatically
+ * powers off the external flash when no longer needed. Otherwise, select the
+ * most basic implementation.
+ */
+#if CONFIG_PM_DEVICE && CONFIG_NORDIC_QSPI_NOR
+ExtFlashHandler sQSPIHandler;
```

Comment:
It's duplicate of `GetOTAImageProcessor` function below. Remove either this or that one.